src/Connection/Cluster/PredisCluster.php

```diff
@@ -55,15 +55,27 @@ class PredisCluster implements ClusterInterface, IteratorAggregate, Countable
      */
     private $connectionParameters;
 
+    /**
+     * @var int
```

Comment:
Would it make sense to add a documentation explaining what is this `readTimeout`?

Reply:
Done